lib/bolt_ext/server.rb

```diff
@@ -71,3 +77,19 @@ class TransportAPI < Sinatra::Base
     [500, "500: Unknown error: #{e.message}"]
   end
 end
+
+class ValidateRequest
```

Comment:
Where should this live? Similarly where should the schemas directory live?

Reply:
I think here is fine - we can always pull it out if it gets too unwieldy